Jacksonstansell

jstansell2 opened the pull request:
This pull request includes answers for Exercise Sets 3 - 6.

jstansell2 committed:
Exercise set 2 from lecture 3.

jstansell2 committed:
Committing Exercise Set 3.

jstansell2 committed:
Committing exercise set 4 from lecture 5.

jstansell2 committed:
Committing csv files corresponding to exercises in Lecture 5.

jstansell2 committed:
Committing exercises for Lecture 6

ougx commented:
In Ex4.7, 
`period = '&period=&begin_date={}&end_date={}'.format(start_date,end_date)` 
should be 
`period = 'period=&begin_date={}&end_date={}'.format(start_date,end_date)`

Please test Your output from `bufferzones` in Ex 6.4

jstansell2 committed:
Commiting Exercise Set 7 - Pandas Basics

jstansell2 committed:
Committing Exercise Set 9.

jstansell2 committed:
Updating Exercise Set 4

jstansell2 committed:
Couldn't see an issue with Exercise Set 6 testing.

jstansell2 commented:
I've added my results for Exercise Set 9 - making wxPython applications. I've included the jupyter notebook file for readability, but to actually run and test the files, the .py files for each question are best. I fixed the suggestion you had for Ex4.7, but couldn't see the problem with Ex6.4 after testing.
